src/utils/isReactComponentMethod.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+import recast from 'recast';
+
+import getPropertyName from './getPropertyName';
+
+const {types: {namedTypes: types}} = recast;
+
+const componentMethods = [
+  'render',
+  'getInitialState',
+  'getDefaultProps',
+  'getChildContext',
+  'componentWillMount',
+  'componentDidMount',
+  'componentWillReceiveProps',
+  'shouldComponentUpdate',
+  'componentWillUpdate',
+  'componentDidUpdate',
+  'componentWillUnmount',
```

Comment:
constructor is removed here https://github.com/reactjs/react-docgen/pull/66/files#diff-04ad70c13ed9a5e61df9714b19011b37R51